### PR TITLE
Opening Post for Rome

### PR DESCRIPTION
**Pull request by tglman**

### CODE REVIEW

File: _posts/2018-08-22-next-stop-rome.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+**Need to convince your boss?** Some talking points: RustFest is the leading non-US based conference focused on Rust and the community around it. We are a community conference with (very) affordable tickets, and we can issue invoices to companies. 
+
+**What is it like?** Talks can be found online for [Berlin](https://www.youtube.com/watch?v=fI4RG_uq-WU&list=PL85XCvVPmGQh8nWR_Z-fTmPGsUWuzb-dn), [Київ](https://www.youtube.com/watch?v=AHprJNUCgQ0&list=PL85XCvVPmGQhvs1Rnet_24B-AI3YSM2YG), [Zürich](https://www.youtube.com/watch?v=jywiVWKm1TI&list=PL85XCvVPmGQj9mqbJizw-zi-EhcpS5jTP) and [Paris](https://www.youtube.com/watch?v=23lRkdDXqY0&list=PL85XCvVPmGQgdqz9kz6qH3SI_hp7Zb4s1). The conference is much more than that though, and you'll have the opportunity to meet people who work on some of the projects and efforts you know and love, share in fine cheese and bread, and drink world famous wine with your new and old friends.
+
+**Want to speak?** The [Call for Proposals](https://cfp.rustfest.eu/) is open! We would love to hear your proposal for a talk, workshop or discussion. Make sure to submit before **Date** As always, we'll be prioritizing talks not yet recorded at Rust conferences.
```
Comment (wolf4ood) [inline _posts/2018-08-22-next-stop-rome.md]: Maybe we should change this, if we are opening CFP later this week

Comment (wolf4ood) [reply _posts/2018-08-22-next-stop-rome.md]: Maybe change also the post name to `2018-08-23-next-stop-rome.md`

File: _posts/2018-08-22-next-stop-rome.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "let rome = rust_fest.next();"
+socialImageSrc: "/assets/posts/rome-panorama.jpg"
+authors:
+  - wolf4ood 
+  - tglman
+---
+They say all the roads lead to Rome, so, after previous successful stops in Berlin, Київ, Zürich and Paris, it was just a matter of time before Rust Fest reached the Italian capital. We hope you'll join us on  **November 24th and 25th, 2018** at [**Congress Center Frentani**](https://www.congressifrentani.it/centrocongressi/), near the main railway station of Rome ([Termini](https://www.romatermini.com/en/)).
```
Comment (badboy) [inline _posts/2018-08-22-next-stop-rome.md]: Remove the comma after "so"

Comment (badboy) [inline _posts/2018-08-22-next-stop-rome.md]:
```patch
- Rust Fest
+ RustFest
```

File: _posts/2018-08-22-next-stop-rome.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "let rome = rust_fest.next();"
+socialImageSrc: "/assets/posts/rome-panorama.jpg"
```
Comment (badboy) [inline _posts/2018-08-22-next-stop-rome.md]: Where's the image coming from and what is its license?

File: _posts/2018-08-22-next-stop-rome.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+authors:
+  - wolf4ood 
+  - tglman
+---
+They say all the roads lead to Rome, so, after previous successful stops in Berlin, Київ, Zürich and Paris, it was just a matter of time before Rust Fest reached the Italian capital. We hope you'll join us on  **November 24th and 25th, 2018** at [**Congress Center Frentani**](https://www.congressifrentani.it/centrocongressi/), near the main railway station of Rome ([Termini](https://www.romatermini.com/en/)).
+
+![The Colosseum](assets/posts/rome.jpg)
```
Comment (badboy) [inline _posts/2018-08-22-next-stop-rome.md]: Where's the image coming from and what is its license?

File: _posts/2018-08-22-next-stop-rome.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+**What is Rustfest again?** It's a two day conference involving one day of workshops, and one day of talks! We aim for a diverse set of talks: technical and non-technical, advanced and beginner friendly content, live coding, panel, or slide based. If you like Rust, the community, or just hacking in general we aim to have something to appeal to you. This year the conference will host up to 350 awesome folks. It occurs bi-annually in a different city each edition.
```
Comment (badboy) [inline _posts/2018-08-22-next-stop-rome.md]: Remove the comma before "and"

File: _posts/2018-08-22-next-stop-rome.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+**Need to convince your boss?** Some talking points: RustFest is the leading non-US based conference focused on Rust and the community around it. We are a community conference with (very) affordable tickets, and we can issue invoices to companies. 
+
+**What is it like?** Talks can be found online for [Berlin](https://www.youtube.com/watch?v=fI4RG_uq-WU&list=PL85XCvVPmGQh8nWR_Z-fTmPGsUWuzb-dn), [Київ](https://www.youtube.com/watch?v=AHprJNUCgQ0&list=PL85XCvVPmGQhvs1Rnet_24B-AI3YSM2YG), [Zürich](https://www.youtube.com/watch?v=jywiVWKm1TI&list=PL85XCvVPmGQj9mqbJizw-zi-EhcpS5jTP) and [Paris](https://www.youtube.com/watch?v=23lRkdDXqY0&list=PL85XCvVPmGQgdqz9kz6qH3SI_hp7Zb4s1). The conference is much more than that though, and you'll have the opportunity to meet people who work on some of the projects and efforts you know and love, share in fine cheese and bread, and drink world famous wine with your new and old friends.
```
Comment (badboy) [inline _posts/2018-08-22-next-stop-rome.md]: cheese and bread, wine -> Pizza, Pasta & Espresso?

File: _posts/2018-08-22-next-stop-rome.md (hunk 1 above)
Comment (badboy) [reply _posts/2018-08-22-next-stop-rome.md]: We definitely need to change this to make clear that the CfP opens soon.
yes about the file name change.

File: _posts/2018-08-22-next-stop-rome.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+**Want to speak?** The [Call for Proposals](https://cfp.rustfest.eu/) is open! We would love to hear your proposal for a talk, workshop or discussion. Make sure to submit before **Date** As always, we'll be prioritizing talks not yet recorded at Rust conferences.
+
+> Note this is a **hard deadline** this year, we will not be able to extend as timelines are already very tight. **Submit early**.
```
Comment (badboy) [inline _posts/2018-08-22-next-stop-rome.md]: Let's remove this for now.

File: _posts/2018-08-22-next-stop-rome.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+**Haven't spoken before but want to?** Submit! It's totally okay! If we accept your talk in our (blind) selection procedure we're happy to be your first experience. If you'd like, we're also happy to mentor you to give you some tips.
+
+**Want to volunteer?** It's great to have a few locals who'd like to help attendees enjoy the experience. If you'd like to help guide a cohort of fellow hackers to your favorite restaurant, give directions, or just generally help out and make a bunch of awesome friends please [let us know here](https://github.com/RustFestEU/blog.rustfest.eu/issues/12).
```
Comment (badboy) [inline _posts/2018-08-22-next-stop-rome.md]: This link is not valid. 
Question is: do we take on official volunteers or do we only offer the space for discussion/planning around group activities?

File: _posts/2018-08-22-next-stop-rome.md (hunk 7 above)
Comment (badboy) [inline _posts/2018-08-22-next-stop-rome.md]: Do we explicitely call out the CfP committee or do we save that for the CfP announcement?

File: _posts/2018-08-22-next-stop-rome.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+**Barriers to participating?** Not all of us can easily attend these events for a variety of reasons. We'll be working with [Diversity Tickets](https://diversitytickets.org/) again. If the ticket cost itself is your only barrier, let us know.
```
Comment (badboy) [inline _posts/2018-08-22-next-stop-rome.md]: Maybe link to the new "infos & policies" page as info for other barriers we try to handle?

File: _data/team.yml [hunk 10]
```diff
@@ -237,3 +237,27 @@ pnkfelix:
...
+  links:
+    - icon: "github"
+      title: "wolf4ood"
+      link: "https://github.com/wolf4ood"
+    - icon: "twitter"
+      title: "wolf4ood"
+      link: "https://twitter.com/wolf4ood"
+tglman:
+  name: "Emanuele Tagliaferri"
+  nick: "tglman"
```
Comment (badboy) [inline _data/team.yml:254]: It's missing a picture for Emanuele.

**Review by badboy — CHANGES_REQUESTED** [review]

File: _posts/2018-08-22-next-stop-rome.md (hunk 4 above)
Comment (wolf4ood) [reply _posts/2018-08-22-next-stop-rome.md]: @badboy 
here it's the source
https://commons.wikimedia.org/wiki/File:Il_Colosseo_(261008409).jpeg

is it ok?

File: _posts/2018-08-22-next-stop-rome.md (hunk 3 above)
Comment (wolf4ood) [reply _posts/2018-08-22-next-stop-rome.md]: @badboy 

here

https://commons.wikimedia.org/wiki/File:Rome_Skyline_(8012016319).jpg

File: _posts/2018-08-22-next-stop-rome.md (hunk 4 above)
Comment (badboy) [reply _posts/2018-08-22-next-stop-rome.md]: > This file is made available under the Creative Commons CC0 1.0 Universal Public Domain Dedication. 

All ok then!

File: _posts/2018-08-22-next-stop-rome.md (hunk 3 above)
Comment (badboy) [reply _posts/2018-08-22-next-stop-rome.md]: > This file is licensed under the Creative Commons Attribution-Share Alike 2.0 Generic license. 
> ...
> attribution – You must attribute the work in the manner specified by the author or licensor (but not in any way that suggests that they endorse you or your use of the work).

We need to name the author then, e.g. at the end of the post with a short note.

File: _posts/2018-08-22-next-stop-rome.md (hunk 6 above)
Comment (wolf4ood) [reply _posts/2018-08-22-next-stop-rome.md]: 👍

File: _posts/2018-08-22-next-stop-rome.md (hunk 8 above)
Comment (wolf4ood) [reply _posts/2018-08-22-next-stop-rome.md]: Not sure. Did we have official volunteers for Paris?

Comment (badboy) [reply _posts/2018-08-22-next-stop-rome.md]: Nope, only the team. We were so many that we didn't know what to do with more people :D

File: _posts/2018-08-22-next-stop-rome.md (hunk 3 above)
Comment (wolf4ood) [reply _posts/2018-08-22-next-stop-rome.md]: This is the social image. i will remove this and use the other one.

File: _posts/2018-08-22-next-stop-rome.md (hunk 8 above)
Comment (wolf4ood) [reply _posts/2018-08-22-next-stop-rome.md]: ok i will remove this content for now. I think we are also good for Rome

File: _posts/2018-08-24-next-stop-rome.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+
+**Haven't spoken before but want to?** Submit! It's totally okay! If we accept your talk in our (blind) selection procedure we're happy to be your first experience. If you'd like, we're also happy to mentor you to give you some tips.
+
+**Barriers to participating?** Not all of us can easily attend these events for a variety of reasons. We'll be working with [Diversity Tickets](https://diversitytickets.org/) again. If the ticket cost itself is your only barrier, let us know. Read more on our [infos & policies](https://rome.rustfest.eu/info/) page.
+
+**Want to help someone else attend? Have a conference budget?** We will be selling supporter tickets as well, with the additional funds being used for inclusivity and diversity efforts. If you can, consider helping our fellow community members.
+
+**Want to help organize?** If you'd like to be part of our organizing team please introduce yourself and let us know at [team@rustfest.eu](mailto:team@rustfest.eu). We prefer it if team members are interested in helping for multiple events, especially if you might be interested in helping bring a RustFest to your area.
+
```
Comment (apiraino) [inline _posts/2018-08-24-next-stop-rome.md]: `We prefer if team members ... `

File: _posts/2018-08-24-next-stop-rome.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+layout: post
+title: "let rome = rust_fest.next();"
+socialImageSrc: "/assets/posts/rome.jpg"
+authors:
+  - wolf4ood 
+  - tglman
+---
+They say all the roads lead to Rome, so after previous successful stops in Berlin, Київ, Zürich and Paris, it was just a matter of time before RustFest reached the Italian capital. We hope you'll join us on  **November 24th and 25th, 2018** at [**Congress Center Frentani**](https://www.congressifrentani.it/centrocongressi/), near the main railway station of Rome ([Termini](https://www.romatermini.com/en/)).
+
+![The Colosseum](assets/posts/rome.jpg)
```
Comment (apiraino) [inline _posts/2018-08-24-next-stop-rome.md]: not sure but I think `/assets/posts/rome.jpg` (leading slash)

**Review by apiraino — APPROVED** [review]

**Review by badboy — APPROVED** [review]

**Pull request merged**